types/handsontable/index.d.ts

```diff
@@ -1,9 +1,99 @@
-// Type definitions for Handsontable 0.30
+// Type definitions for Handsontable 0.31
 // Project: https://handsontable.com/
-// Definitions by: Handsoncode sp. z o.o. <http://handsoncode.net/>
+// Definitions by: Handsoncode sp. z o.o. <http://handsoncode.net/>, Ryan Riley <https://github.com/panesofglass>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped\
 
-declare namespace ht {
+declare namespace __Handsontable {
+    // The Core class is used inside the Handsontable namespace in the actual source code.
+    // However, there is a resolution issue when a class outside a namespace or module
+    // extends a class inside a module with the same name. That issue includes a work-around.
+    // Using the workaround found here: https://github.com/Microsoft/TypeScript/issues/3602
```

Comment:
Explanatory comment related to the use of two namespaces. I couldn't find a more recent resolution to the referenced issue.